meta-ostro/classes/image-dsk.bbclass

```diff
@@ -38,6 +39,74 @@ inherit pythonnative
 # suffix. Probably should be moved to ostro-os.conf eventually.
 IMAGE_NAME_SUFFIX = ""
 
+create_ova() {
+
+  if [ "x${MACHINE}x" != "xintel-corei7-64x" ]; then
+    exit 0
+  fi
+
+  # 64 bit linux with sata bus
+  OS_TYPE="Linux_64"
+  STORAGE_NAME="SATA"
+  STORAGE_BUS_TYPE="sata"
+
+  VM_NAME="${IMAGE_NAME}"
+  # 512 should be large enough value to run Ostro (Galileo has 256MB RAM for example)
+  # but small enough value that even a modest host machine can afford it
+  RAM="512"
+  # min allowed - as we do not run graphical environment, it should be enough
+  VRAM="12"
+  # As above with RAM, 2 cores should be enough for Ostro without crippling the host machine
+  CPU_CORE_COUNT=2
+  # required for successful boot
+  FIRMWARE="efi"
+  IOAPIC="on"
+
+  # default settings for hard drive
+  STORAGE_PORT="0"
+  STORAGE_DEVICE="0"
+  STORAGE_DEVICE_TYPE="hdd"
+
+  
+  RAW_IMAGE="${DEPLOY_DIR_IMAGE}/${IMAGE_NAME}.dsk"
+  VIRTUALBOX_IMAGE="${DEPLOY_DIR_IMAGE}/${IMAGE_NAME}.vmdk"
```

Comment:
I just used the approach described here: https://www.virtualbox.org/manual/ch09.html#rawdisk

Looking at the available VBoxManage commands, converthd with -dstformat VDI might be more appropriate though.

However, if the vdi compression (conversion?) already creates a vdi file, and appliance creation creates a vdi file from the raw image, wouldn't this cause issues? At very least this step would be duplicated.